### PR TITLE
Map info windows

### PR DESCRIPTION
**Pull request by aroquev00**
The map's markers now display an Info Window with a small description text.

Deployment: https://aroquev-step-2020.uc.r.appspot.com/

### CODE REVIEW

**Comment by seanjohnite** [comment]
Looks good! I can't seem to trigger the info boxes on the deployed version? I click and nothing happens...

**Comment by aroquev00** [comment]
Ohh it's not deployed yet, let me merge the last approved PR and I'll deploy it so you can see it!

File: aroquev/pom.xml [hunk 1]
```diff
@@ -48,6 +48,7 @@
         <configuration>
           <deploy.projectId>aroquev-step-2020</deploy.projectId>
           <deploy.version>1</deploy.version>
+          <port>8181</port>
```
Comment (brettapeters) [inline aroquev/pom.xml:51]: Is this port just for dev?

Comment (aroquev00) [reply aroquev/pom.xml]: Yup, had a port conflict in my Cloudtop

File: aroquev/src/main/webapp/script.js [hunk 2]
```diff
@@ -472,9 +472,259 @@ function submitComment(commentForm, loginInfo) {
   });
 }
 
-/** Creates a map and adds it to the page. */
+
+
+let map;
+let markers = [];
```
[the remark this answers is not in the document]
Comment (aroquev00) [reply aroquev/src/main/webapp/script.js]: True, removed.

**Comment by aroquev00** [comment]
The changes of this PR are now live! https://aroquev-step-2020.uc.r.appspot.com/

**Pull request merged**